cider/function/CMakeLists.txt

```diff
@@ -17,12 +17,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
+include_directories(${CMAKE_SOURCE_DIR}/thirdparty/velox)
```

Comment:
please don't include substrait directory here, they should already included in the top CMake file. If you really need include here, you should include either substrait in velox or substrait in thirdparty depend on whether enable velox build.

Reply:
Done.